analyze_logs/src/analyzer.py

```diff
@@ -58,8 +66,7 @@ def fullAnalysis(self):
             e = sys.exc_info()[0]
             self.log.error("Could not process: " + event.message_text + " because %s" % e)
             traceback.print_exc(file=sys.stdout)
-      else:
-        break 
+      time.sleep(self.sleep_time)   
```

Comment:
this doesn't seem to relate to the commit message - which says don't exit rather than add a sleep